recipes/mirotalk-bro/package.json

```diff
@@ -0,0 +1,12 @@
+{
+	"id": "mirotalk-bro",
```

Comment:
please fix indentation - we prefer 2 spaces

Reply:
these are still using 4 spaces - please convert to 2 spaces (this is also specified/configured in the `.editorconfig` file at the root of the repo.